### ISSUE TITLE
document ace-based shortcuts

### ISSUE THREAD

**Issue by gordonwoodhull** [att/rcloud#1870]
We have a few keyboard commands you can find by grepping for `commands.addCommands`, e.g. the stuff that runs the current cell or copies the selection to a new cell and runs it.

We should document this through the new shortcuts facility, even though the keys are not actually implemented there.

I don't think there's any need to document Ace's own shortcuts, but maybe we should [link to them](https://github.com/ajaxorg/ace/wiki/Default-Keyboard-Shortcuts).

**Comment by shaneporter** [comment]
I'm mostly done but I have two things to mention:

1. Linking to the Ace shortcuts is ok I guess, but it would be a relatively quick copy/paste/edit exercise to add them. This would benefit the user since they wouldn't navigate away from RCloud to find out about them, _but_:
2. The increase in number of shortcuts would mean that the shortcuts dialog would benefit from a tabs by category appoach.

**Comment by shaneporter** [comment]
Actually, it seems as though not all of [those shortcuts](https://github.com/ajaxorg/ace/wiki/Default-Keyboard-Shortcuts) are active. I'm looking at you, Multicursor.

**Comment by gordonwoodhull** [comment]
I think many of the ace keyboard shortcuts are governed by the options we set - grep for `widget.setOptions` and then take a look at [Configuring Ace](https://github.com/ajaxorg/ace/wiki/Configuring-Ace). We've only pulled stuff in on demand, so there are probably more useful features that we're not enabling.

Note: our ace code customized, especially around R support, which we pull from RStudio and their [ace-cpp-autoindent](https://github.com/rstudio/ace-cpp-autoindent), which is closely related to their R mode. Also some autocomplete stuff and probably other things.
